src/main/java/org/jahia/modules/newsletter/action/SubscribeAction.java

```diff
@@ -238,6 +260,31 @@ private boolean sendConfirmationMail(JCRSessionWrapper session, String email, JC
         return false;
     }
 
+	/**
+	* Check if templatSet has mail template
+	*/
+    private String getTemplateName(String template, JCRNodeWrapper node,  final Locale locale, String defaultTemplate){
+    	String templateToReturn = defaultTemplate;
+
+    	try {
+	    	//try if it is multilingual
```

Comment:
This code is duplicate from unsubscribeAction.
We need to centralize this code in one unique place.